specification.md

```diff
@@ -0,0 +1,1199 @@
+    <td>The mime type of the image.  If not provided, the format may be guessed from the extension or the Content-Type header of the resource.</td>
+    <td>"image/png"</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+</table>
+
+
+### video
+
+Represents a video submitted by the Contact
+
+#### Response format
+
+A string with the URL where the video can be retrieved. (TODO: Do we want to support inline video data?)
+
+```
+"https://myexampleflowserver.com/resources/videos/23429837434.png"
+```
+
+#### Type options (type_options)
+
+<table>
+  <tr>
+    <td>Object</td>
+    <td>Required</td>
+    <td>Details</td>
+    <td>Example</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+</table>
+
+
+#### Response Metadata
+
+<table>
+  <tr>
+    <td>Object</td>
+    <td>Required</td>
+    <td>Details</td>
+    <td>Example</td>
+  </tr>
+  <tr>
+    <td>`format`</td>
+    <td>Recommended.</td>
+    <td>The mime type of the video.  If not provided, the format may be guessed from the extension or the Content-Type header of the resource.</td>
+    <td>"video/mp4"</td>
+  </tr>
+  <tr>
+    <td>`language`</td>
+    <td>Optional</td>
+    <td>The ISO 639-3 code for the language of the response, if known.</td>
+    <td>{'language':'eng'}</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+</table>
+
+
+### audio
+
+Represents an audio recording submitted by the Contact
+
+#### Response format
+
+A string with the URL where the audio can be retrieved. (TODO: Do we want to support inline audio data?)
+
+```
+"https://myexampleflowserver.com/resources/audio/23429837435.png"
+```
+
+#### Type options (type_options)
+
+<table>
+  <tr>
+    <td>Object</td>
+    <td>Required</td>
+    <td>Details</td>
+    <td>Example</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+</table>
+
+
+#### Response Metadata
+
+<table>
+  <tr>
+    <td>Object</td>
+    <td>Required</td>
+    <td>Details</td>
+    <td>Example</td>
+  </tr>
+  <tr>
+    <td>`format`</td>
+    <td>Recommended.</td>
+    <td>The mime type of the audio.  If not provided, the format may be guessed from the extension or the Content-Type header of the resource.</td>
+    <td>"audio/wav"</td>
+  </tr>
+  <tr>
+    <td>`language`</td>
+    <td>Optional</td>
+    <td>The ISO 639-3 code for the language of the response, if known.</td>
+    <td>{'language':'eng'}</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+</table>
+
+
+### geo_point
+
+Represents a geospatial coordinate on the surface of the earth.
+
+#### Response format
+
+An array of two floating point numbers with the latitude and longitude: `[lat, long]`
+
+```
+[52.0835780,-106.6104880]
```

Comment:
Should be simple to support those 4-tuples as well.  Should GeoJSON objects be a separate question type?  I think it's helpful to know that something is specifically a point, rather than any general form of path/boundary.